test/graphql/queries/profile_query_test.rb

```diff
@@ -91,10 +91,10 @@ class ProfileQueryTest < ActiveSupport::TestCase
 
     assert_equal profiles(:one).id,
                  result['data']['profile']['policy']['id']
-    assert_equal profiles(:one).name,
-                 result['data']['profile']['policy']['name']
     assert_equal profiles(:one).ref_id,
                  result['data']['profile']['policy']['refId']
+    assert_equal policies(:one).name,
```

Comment:
Would it make sense to also `update!` the name and description just to verify that the updated values are actually used?

Reply:
I guess no need, as the policy has name "policy1" and profile has "profile1". Or, did I missed something?